analyzer/codechecker_analyzer/analyzers/clangsa/analyzer.py

```diff
@@ -596,6 +613,39 @@ def construct_config_handler(cls, args):
             cmdline_checkers,
             'enable_all' in args and args.enable_all)
 
+        # If --stats flag is provided then enable statistics-based checkers
+        # unless explicitly disabled by the user.
+        if 'stats_enabled' in args:
+            # In some Clang versions these checkers may be in some checker
+            # group (e.g. alpha...), so let's find the actual checker name
+            # first.
+            special_return_value = next((
+                checker for checker, _ in checkers
+                if SpecialReturnValueCollector.checker_analyze in checker),
+                None)
+            unchecked_return_value = next((
+                checker for checker, _ in checkers
+                if ReturnValueCollector.checker_analyze in checker),
+                None)
+
+            if special_return_value and not _is_user_disabled_checker(
+                    special_return_value,
+                    cmdline_checkers):
+                handler.set_checker_enabled(special_return_value)
+
+            if unchecked_return_value and not _is_user_disabled_checker(
+                    unchecked_return_value,
+                    cmdline_checkers):
+                handler.set_checker_enabled(unchecked_return_value)
+
+        # Statistics collector checkers must be explicitly disabled
+        # as they trash the output.
```

Comment:
You should also state that their job was done anyway in a preceding analysis phase. Otherwise I enjoy the sh!ttalk :)